src/fastpm.c

```diff
@@ -1740,6 +1752,15 @@ write_powerspectrum(FastPMSolver * fastpm, FastPMForceEvent * event, RunData * p
             fastpm_path_ensure_dirname(CONF(prr->lua, write_powerspectrum));
             fastpm_powerspectrum_write(&ps, buf, event->N);
         }
+        if(CONF(prr->lua, ncdm_linearresponse)) {
+            sprintf(buf, "%s_nu_%0.04f.txt", CONF(prr->lua, write_powerspectrum), event->a_f);
+            fastpm_info("writing neutrino power spectrum to %s\n", buf);
+            /* Need to change the power spectrum normalisation here for saving the output power spectrum.*/
+            const double MtotbyMcdm = fastpm->cosmology->Omega_m/(fastpm->cosmology->Omega_m - pow(event->a_f,3)*Omega_ncdmTimesHubbleEaSq(event->a_f, fastpm->cosmology))/pow(HubbleEa(event->a_f, fastpm->cosmology),2);
```

Comment:
The RHS reads as $\frac{\Omega_m}{\Omega_m - a^3\Omega_\nu(a)E^2(a)}\cdot\frac{1}{E^2(a)}$. Should it be $\frac{\Omega_m}{\Omega_m - a^3\Omega_\nu(a)E^2(a)/E^2(a)} = 1/f_c$?

Reply:
Possibly! But also I think we don't need the scaling factor at all.